Blog/style/style.css

```
@@ -0,0 +1,52 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 0;
+  list-style: none;
+  font-size: "Inter", "Space Grotesk", "sans";
+}
+:root{
+  --dark-10: #121214;
+  --dark-20: #17171A;
+  --dark-30: #202024;
+  --dark-40: #252529;
+  --dark-50: #AFABB6;
+  --dark-60: #FFFFFF;
+}
```

Sugestão: colocar uma quebra de linhas entre o fechamento da tag e .container{

```suggestion
}

body{
```